examples/ultralytics-yolo/deepsparse_utils.py

```diff
@@ -514,6 +514,15 @@ def yolo_onnx_has_postprocessing(model_path: str) -> bool:
     return all(num_dims > outputs_num_dims[0] for num_dims in outputs_num_dims[1:])
 
 
+def _download_model_if_stub(path: str) -> str:
```

Comment:
this function is used outside the scope of this file, so let's remove the starting `_` from the name and also add this to the `__all__` list.  Additionally, let's use this at the start of `modify_yolo_onnx_input_shape` since it's the same code

Reply:
Resolved! ⭐